src/components/CreateRouteModal/CreateRouteModal.jsx

```diff
@@ -0,0 +1,159 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React, { useState, useEffect } from 'react';
+import { PropTypes } from 'prop-types';
+import { useForm } from 'react-hook-form';
+import * as yup from 'yup';
+import { yupResolver } from '@hookform/resolvers/yup';
+import {
+  Box,
+  Button,
+  FormControl,
+  FormLabel,
+  HStack,
+  Heading,
+  Input,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  Select,
+} from '@chakra-ui/react';
+import { SingleDatepicker } from 'chakra-dayzed-datepicker';
+import { calendarConfigs } from '../../utils/utils';
+import { createRoute, getDrivers } from '../../utils/RouteUtils';
```

Comment:
🚫 **[eslint]** <[import/extensions](https://github.com/import-js/eslint-plugin-import/blob/v2.27.5/docs/rules/extensions.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing file extension for "../../utils/RouteUtils"